src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -100,34 +86,42 @@ class OrganizationReleases extends React.Component {
     );
   }
 
-  render() {
-    if (!new Set(this.props.organization.features).has('sentry10')) {
-      return this.renderNoAccess();
-    }
+  renderError() {
+    return this.renderBody();
+  }
+
+  renderLoading() {
+    return this.renderBody();
+  }
 
+  renderBody() {
     return (
-      <Content>
-        <Header>
-          <div>
+      <Feature
+        features={['organizations:sentry10']}
+        organization={this.props.organization}
+        renderDisabled={this.renderNoAccess}
```

Comment:
We should use the default `renderDisabled`, which will be "Coming soon". This way, I think we'll also be able to customize the disabled message thrigh getsentry via hooks.